### DISCUSSION TITLE
Stock status values

### DISCUSSION THREAD

**Discussion by anonkwe** [inventree/InvenTree#5111] [Q&A]
Hi, I changed some stock status values for custom ones, on the stockitem page, it displays okay but on the stockitem tables by location, it still gets the original value that is displayed. ex:

value that shoudl be displayed: 
![image](https://github.com/inventree/InvenTree/assets/61838766/3dcf41be-ec95-4ff3-87dc-4bbd641c4568)

value displayed on the table:

![image](https://github.com/inventree/InvenTree/assets/61838766/314359cd-8936-45d1-94da-1fcdba698812)

Same thing in stock tracking info:

![image](https://github.com/inventree/InvenTree/assets/61838766/ba58eb9d-7550-43da-9012-c801c42e2a23)

any idea?

**Comment by matmair** [comment]
@anonkwe how did you change them? Custom states are not implemeted yet #4969

**Comment by anonkwe** [comment]
@matmair through status_codes.py and django files. I understand it's not implemented yet but my question is where are those tables taking their values at?

**Reply by matmair** [reply]
I do not recommend changing the source files itself - will lead to problems. You will need to prerender all js files with the management command.

**Comment by anonkwe** [comment]
@matmair  i understand that it can lead to problems, im testing some stuff tho. what do you mean by prerender all js files with the management command.
